### ISSUE TITLE
unable to livestream

### ISSUE THREAD

**Issue by Lvortex777** [WWBN/AVideo#5468]
when i attempt to go live it saids livestream failed, and using obs its saids there something wrong with rtmp settings

**Comment by Lvortex777** [comment]
![Screenshot 2021-07-23 163211](https://user-images.githubusercontent.com/86401367/126843805-a7681b19-f12c-47dd-988e-c2ccc134735c.png)

**Comment by Lvortex777** [comment]
it  keeps doing the same thing

**Comment by DanielnetoDotCom** [comment]
can you go live using OBS?
https://github.com/WWBN/AVideo/wiki/How-to-make-a-live-stream#from-pc

**Comment by Lvortex777** [comment]
yes i see what you was saying it was actually cloudflare blocking the stream so i removed the cdn .
